Apps/Inc/Tasks.h

```diff
@@ -17,8 +17,7 @@
 
 /**
  * Priority Definitions
- */
-#define TASK_FAULT_STATE_PRIO               1
+ */ 
 #define TASK_INIT_PRIO                      2
 #define TASK_READ_TRITIUM_PRIO              3
 #define TASK_SEND_TRITIUM_PRIO              4
```

Comment:
Interesting point- I never thought of that! My guess is that they're not an enum because they're not needed frequently in application code, but maybe @Cam0Cow or someone else could tell us more?

Reply:
No reason really, these are usually done as `#define`, but it's fine either way.